tests/test_bake_project.py

```diff
@@ -181,7 +181,7 @@ def check_quality(result):
         pylintrc = str(result.project.join('pylintrc'))
         for filename in filenames:
             name = os.path.join(dirpath, filename)
-            if not name.endswith('.py'):
+            if not name.endswith('.py') or filename == 'conf.py':
```

Comment:
@pomegranited This test runs quality checks on the generated project, including pep8, pylint and so on.

The particular issue I was not able to fix is that (watch the hands :))

1. It uses `pylint` in tox environment at say ~/cookiecutter-django-app/.tox/py27/bin/pylint
2. ... to test code in the generated project at something like /tmp/something/something/your-project-title-gors-here/docs/conf.py
3. ... which contains local import in form of `import your_project_goes_here`
4. ... which works perfectly fine if you run pylint on the generated project *from* the project itself
5. ... but fails pylint if it is invoked as in step 1, because it can't find `your_project_title_goes_here` module as it is not in the pylint's current working directory.

So, I just decided that it would be simpler to just exclude the file, especially since it's not even linted by the actual generated project.